src/lib.rs

```diff
@@ -3638,6 +3703,22 @@ fn command_add_output_file(
     }
 }
 
+#[derive(Clone, Copy)]
+enum AppleOs {
+    MacOs,
+    Ios,
+    WatchOs,
+}
+impl Display for AppleOs {
```

Comment:
This looks like it's only used in error messages, no? Assuming that's right, can it be a `Debug` impl to hint that it's not semantically important?

Reply:
Swapped it out.